DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.address`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.address` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.automotive`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.automotive` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.bank`.
DEBUG    faker.factory:factory.py:104 Specified locale `en_US` is not available for provider `faker.providers.bank`. Locale reset to `en_GB` for this provider.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.barcode`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.barcode` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.color`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.color` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.company`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.company` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.credit_card`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.credit_card` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.currency`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.currency` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.date_time`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.date_time` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:115 Provider `faker.providers.file` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.geo`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.geo` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.internet`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.internet` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:115 Provider `faker.providers.isbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.job`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.job` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.lorem`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.lorem` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.misc`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.misc` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.person`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.person` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.phone_number`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.phone_number` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:115 Provider `faker.providers.profile` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:115 Provider `faker.providers.python` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.ssn`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.ssn` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:115 Provider `faker.providers.user_agent` does not feature localization. Specified locale `en_US` is not utilized for this provider.
INFO     root:conftest.py:21 -----------正在执行测试用例开始的准备工作,打开浏览器-----------
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/window/maximize {}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/window/maximize HTTP/1.1" 200 51
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/window/maximize {}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/window/maximize HTTP/1.1" 200 51
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/timeouts {"pageLoad": 60000}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/timeouts HTTP/1.1" 200 14
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/refresh {}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/refresh HTTP/1.1" 200 14
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/url {"url": "https://www.baidu.com/"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/url HTTP/1.1" 200 14
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/timeouts {"implicit": 10000}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/timeouts HTTP/1.1" 200 14
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
INFO     root:webpage.py:28 打开网页：https://www.baidu.com/
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/element {"using": "xpath", "value": "//*[@id=\"kw\"]"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/element HTTP/1.1" 200 88
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/element/8408b3be-cc1c-4990-a47f-c3e8f4367fba/clear {"id": "8408b3be-cc1c-4990-a47f-c3e8f4367fba"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/element/8408b3be-cc1c-4990-a47f-c3e8f4367fba/clear HTTP/1.1" 200 14
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
INFO     root:webpage.py:48 清空文本
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/element/8408b3be-cc1c-4990-a47f-c3e8f4367fba/value {"text": "\u6d4b\u8bd5", "value": ["\u6d4b", "\u8bd5"], "id": "8408b3be-cc1c-4990-a47f-c3e8f4367fba"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/element/8408b3be-cc1c-4990-a47f-c3e8f4367fba/value HTTP/1.1" 200 14
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
INFO     root:webpage.py:50 输入文本:测试
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/element {"using": "xpath", "value": "//*[@id=\"su\"]"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/element HTTP/1.1" 200 88
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 POST http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623/element/76513a92-2342-4ca5-bfce-6411f91edcaf/click {"id": "76513a92-2342-4ca5-bfce-6411f91edcaf"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "POST /session/6914ef236e078f8d8ba622342dfed623/element/76513a92-2342-4ca5-bfce-6411f91edcaf/click HTTP/1.1" 200 14
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
INFO     root:webpage.py:55 点击元素:('xpath', '//*[@id="su"]')
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:412 DELETE http://localhost:7703/session/6914ef236e078f8d8ba622342dfed623 {}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://localhost:7703 "DELETE /session/6914ef236e078f8d8ba622342dfed623 HTTP/1.1" 200 14
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:463 Finished Request
INFO     root:conftest.py:27 -----------关闭浏览器-----------